### PR TITLE
add double quotes in curl urls

### PR DESCRIPTION
**Pull request by sirineREKIK**
**This PR closes the issue [#8978](https://github.com/IQSS/dataverse/issues/8978)** 
(Dev by INRAE)

### CODE REVIEW

File: doc/sphinx-guides/source/api/native-api.rst [hunk 1]
```diff
@@ -56,13 +56,13 @@ Next you need to figure out the alias or database id of the "parent" Dataverse c
   export SERVER_URL=https://demo.dataverse.org
   export PARENT=root
 
-  curl -H X-Dataverse-key:$API_TOKEN -X POST $SERVER_URL/api/dataverses/$PARENT --upload-file dataverse-complete.json
+  curl -H "X-Dataverse-key:$API_TOKEN" -X POST "$SERVER_URL/api/dataverses/$PARENT --upload-file dataverse-complete.json"
```
Comment (donsizemore) [inline doc/sphinx-guides/source/api/native-api.rst:59]: should the close-quote here follow $PARENT? (same question applies to other similar changes below)

Comment (sirineREKIK) [reply doc/sphinx-guides/source/api/native-api.rst:59]: Hello @donsizemore ,
Thank you for this return.
Yes you are right, the close-quote here should follow $PARENT,  it's a mistake on my side. 
I will fix that next Wednesday.

Comment (donsizemore) [reply doc/sphinx-guides/source/api/native-api.rst:59]: @sirineREKIK wonderful, thank you! there are more such instances below in the page.

Comment (sirineREKIK) [reply doc/sphinx-guides/source/api/native-api.rst]: Hello @donsizemore 
I pushed a new commit to fix mistakes.
Regards,

**Review by pdurbin — COMMENTED** [review]
@sirineREKIK can you please resolve the merge conflicts?

**Review by donsizemore — APPROVED** [review]
I didn't test every one of these(!) but from my ocular examination, minus a few pre-existing spacing issues, this looks great. Thank you @sirineREKIK!

File: doc/sphinx-guides/source/api/native-api.rst [hunk 2]
```diff
@@ -265,13 +265,13 @@ To clear the metadata blocks set by a parent collection, submit an empty array (
   export SERVER_URL=https://demo.dataverse.org
   export ID=root
 
-  curl -H X-Dataverse-key:$API_TOKEN" -X POST -H "Content-type:application/json" $SERVER_URL/api/dataverses/$ID/metadatablockfacets --upload-file metadata-block-facets.json
+  curl -H "X-Dataverse-key:$API_TOKEN" -X POST -H "Content-type:application/json" "$SERVER_URL/api/dataverses/$ID/metadatablockfacets" --upload-file metadata-block-facets.json
```
Comment (pdurbin) [inline doc/sphinx-guides/source/api/native-api.rst:268]: Good catch. Weird that this was unbalanced.

File: doc/sphinx-guides/source/api/native-api.rst [hunk 3]
```diff
@@ -658,13 +658,13 @@ To import a dataset with an existing persistent identifier (PID), you have to pr
...
 
-  curl -H X-Dataverse-key:$API_TOKEN -X POST $SERVER_URL/api/dataverses/$DATAVERSE_ID/datasets/:importddi?pid=$PERSISTENT_IDENTIFIER&release=yes --upload-file ddi_dataset.xml
+  curl -H "X-Dataverse-key:$API_TOKEN" -X POST "$SERVER_URL/api/dataverses/$DATAVERSE_ID/datasets/:importddi?pid=$PERSISTENT_IDENTIFIER&release=yes" --upload-file ddi_dataset.xml
 
 The fully expanded example above (without environment variables) looks like this:
 
 .. code-block:: bash
 
-  curl -H X-Dataverse-key:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx -X POST https://demo.dataverse.org/api/dataverses/root/datasets/:importddi?pid=doi:ZZ7/MOSEISLEYDB94&release=yes --upload-file ddi_dataset.xml
+  curl -H "X-Dataverse-key:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" -X POST "https://demo.dataverse.org/api/dataverses/root/datasets/:importddi?pid=doi:ZZ7/MOSEISLEYDB94&release=yes" --upload-file ddi_dataset.xml
```
Comment (pdurbin) [inline doc/sphinx-guides/source/api/native-api.rst:667]: Ha nice MOSEISLEY. Nice.

**Review by pdurbin — APPROVED** [review]
Phew! Long PR but it looks right to me! I scrolled through the diff and checked to make sure the page it self built properly: https://dataverse-guide--9722.org.readthedocs.build/en/9722/api/native-api.html

Approved.

**Pull request merged**